src/transformers/models/bit/modeling_bit.py

```diff
@@ -737,7 +824,7 @@ class BitForImageClassification(BitPreTrainedModel):
     def __init__(self, config):
         super().__init__(config)
         self.num_labels = config.num_labels
-        self.bit = BitModel(config)
+        self.resnetv2 = BitModel(config)
```

Comment:
You can change all "resnetv2" to "bit", as I updated that

Reply:
You mean you updated the conversion script accordingly?

Reply:
[232c22d](https://github.com/NielsRogge/transformers/pull/48/commits/232c22d925f155fc43882f4cb9cd3b6ea5d7d9e5)